intelhex/__init__.py

```diff
@@ -578,47 +582,6 @@ def write_hex_file(self, f, write_start_addr=True, eolstyle='native'):
             # Python 2
             table = ''.join(chr(i).upper() for i in range_g(256))
 
-        # start address record if any
```

Comment:
Moving this block to the end has nothing to do with this pull request and also breaks the output for many tools.